### PR TITLE
Hierarchy migration

### PR DESCRIPTION
**Pull request by danielmarbach**
Introduces a sample that demonstrates a live migration from on topic to another

### CODE REVIEW

**Comment by danielmarbach** [comment]
Requires https://github.com/Particular/docs.particular.net/pull/5958 to compile

File: samples/azure-service-bus-netstandard/hierarchy-migration/sample.md [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+include: asb-connectionstring-xplat
+
+## Code walk-through
+
+This sample demonstrates how an endpoint can be "life" migrated from one topic to another:
+
+* `Endpoint1` publishes `Event1` messages and subscribes to `Event2` messages.
+* `Endpoint2` subscribes to `Event1` messages and publishes `Event2` messages.
+* `Migration` does a step by step life migration of `Endpoint2` while the endpoints are running and exchanging messages with each other.
```
Comment (danielmarbach) [inline samples/azure-service-bus-netstandard/hierarchy-migration/sample.md:22]:
```suggestion
* `Migration` does a step by step live migration of `Endpoint2` while the endpoints are running and exchanging messages with each other. 
```

**Review by lailabougria — COMMENTED** [review]
Here are some small suggestions. I have a few questions about the sample, but it's probably most effective if we can quickly discuss them on a call instead.

File: samples/azure-service-bus-netstandard/hierarchy-migration/ASBS_3/Migration/Visualize.cs [hunk 2]
```diff
@@ -0,0 +1,546 @@
...
+            --> samples.asbs.hierarchymigration.endpoint2.migration
+
+       samples.asbs.hierarchymigration.endpoint1
+       samples.asbs.hierarchymigration.endpoint2
+       samples.asbs.hierarchymigration.endpoint2.migration
+     */
+    public static void Endpoint2SubscriptionUnderSubscriptionBundleSetup()
+    {
+        var sectionDescription =
+            new Rule($"The subscription {Hierarchy.Endpoint2SubscriptionName} on topic {Hierarchy.SubscriptionBundleName} has been setup")
```
[the remark this answers is not in the document]
Comment (danielmarbach) [reply samples/azure-service-bus-netstandard/hierarchy-migration/ASBS_3/Migration/Visualize.cs:244]: Yeah I agree. Will adjust. I will use `'name'` as a pattern

**Pull request merged**